## white_all
#parameter_matrix: [0.441857, 0.194309, 0.014962, 0.262786, 0.055741, -0.020419, -0.022415, -0.069618, -0.025812, -0.068025, 0.046434, 0.009697, 0.273485, -0.101651, -0.193611, 0.09931, 0.027246, 0.064093, -0.02004, 0.040417, 0.195477, 0.26279, 0.199995, 0.156124, -0.068131, -0.01717, 0.043594, 0.037828, -0.039724, 0.04242, -0.040241, 0.104992, -0.004045, -0.045852, 0.000886, 0.119949, -0.041414, -0.031871, 0.021606, 0.043291, 0.053849, 0.267678, 0.514346, 0.064546, -0.178531, -0.039248, 0.267648, -0.002456]
## white average
#parameter_matrix: [0.401842, 0.201991, 0.033787, 0.315455, 0.019466, -0.003106, 0.001229, -0.050173, -0.03922, -0.048674, 0.045102, -0.001201, 0.180402, -0.076363, -0.1522, 0.113727, -0.027084, 0.062803, 0.008172, 0.027923, 0.198572, 0.297825, 0.247632, 0.21666, -0.079975, 0.00309, 0.079896, 0.017928, -0.120956, 0.050854, 0.0007, 0.106772, -0.034398, -0.045716, 0.039336, 0.153742, -0.073722, -0.030142, 0.050402, 0.044603, 0.05458, 0.241364, 0.466982, 0.101796, -0.222205, -0.059614, 0.271009, 0.070987]
## white average2
#parameter_matrix: [0.33605, 0.177658, 0.074484, 0.317854, 0.004651, -0.035876, -0.006386, 0.00846, -0.034742, -0.051705, 0.052528, 0.020423, 0.095515, -0.112181, -0.083653, 0.148169, -0.065279, -0.008904, 0.016536, 0.0551, 0.207049, 0.25537, 0.252751, 0.285891, -0.063151, -0.01501, 0.086999, 0.037629, -0.152494, -0.060756, 0.045043, 0.156693, -0.074916, -0.036329, 0.124881, 0.140599, -0.090893, -0.036771, 0.072742, 0.050473, 0.08489, 0.233129, 0.378704, 0.080155, -0.237328, -0.040839, 0.320753, 0.053174]
## white average3
parameter_matrix: [0.388954, 0.185984, 0.044723, 0.29032, 0.030196, -0.028147, -0.0144, -0.030579, -0.030277, -0.059865, 0.049481, 0.01506, 0.1845, -0.106916, -0.138632, 0.12374, -0.019017, 0.027595, -0.001752, 0.047759, 0.201263, 0.25908, 0.226373, 0.221008, -0.065641, -0.01609, 0.065297, 0.037729, -0.096109, -0.009168, 0.002401, 0.130842, -0.03948, -0.04109, 0.062884, 0.130274, -0.066153, -0.034321, 0.047174, 0.046882, 0.06937, 0.250404, 0.446525, 0.072351, -0.20793, -0.040044, 0.2942, 0.025359]
## white_select 2
#parameter_matrix: [0.361827, 0.209673, 0.052613, 0.368124, -0.016808, 0.014207, 0.024873, -0.030727, -0.052628, -0.029323, 0.043769, -0.012099, 0.087319, -0.051075, -0.110789, 0.128143, -0.081414, 0.061512, 0.036384, 0.01543, 0.201666, 0.33286, 0.295269, 0.277196, -0.091819, 0.023351, 0.116198, -0.001972, -0.202189, 0.059287, 0.041642, 0.108552, -0.064751, -0.04558, 0.077786, 0.187535, -0.10603, -0.028413, 0.079199, 0.045915, 0.05531, 0.21505, 0.419619, 0.139047, -0.265878, -0.07998, 0.27437, 0.144431]
## white_select
#parameter_matrix: [0.341452, 0.185904, 0.054128, 0.327987, -0.002406, -0.0187, 0.00048, -0.009839, -0.044652, -0.052902, 0.057387, 0.016062, 0.08418, -0.108305, -0.085742, 0.160211, -0.065307, 0.016092, 0.034637, 0.01379, 0.235386, 0.31467, 0.29868, 0.268964, -0.059949, 0.003956, 0.117799, 0.029103, -0.164547, -0.044134, 0.057876, 0.108827, -0.06543, -0.034734, 0.109653, 0.175008, -0.099957, -0.035658, 0.064806, 0.05048, 0.086759, 0.251636, 0.384472, 0.057945, -0.23993, -0.047053, 0.281978, 0.091245]
## gray_select
#parameter_matrix: [0.346111, 0.209221, 0.06558, 0.36401, -0.001782, 0.01301, 0.010571, -0.018581, -0.044622, -0.02238, 0.040362, -0.010768, 0.086042, -0.037891, -0.112049, 0.115037, -0.065992, 0.044246, 0.02612, 0.028847, 0.216661, 0.311887, 0.283727, 0.310446, -0.081173, 0.021715, 0.11139, 0.003431, -0.180704, 0.03433, 0.039571, 0.094851, -0.061921, -0.035213, 0.086707, 0.161487, -0.097591, -0.030443, 0.074982, 0.040241, 0.075414, 0.21296, 0.395104, 0.158364, -0.242331, -0.062388, 0.275515, 0.113586]
